src/components/generic/Select/Select.jsx

```diff
@@ -67,6 +79,7 @@ export default function Select({
         tabIndex={tabIndex}
         title={tooltip}
         aria-label={`${label ? `${label}. ` : ''}${ariaLabel}`}
+        onKeyDown={handleKeyDown}
```

Comment:
as the change is practically done on the `Select` element, please update the commit message:
> feat(Select): toggle/collapse dropdown when pressing enter or space on the keyboard